.github/workflows/high-impact-alert.js

```diff
@@ -13,9 +13,12 @@ const main = async (params) => {
     }
 
     const { html_url, number, title } = context.payload.pull_request;
-    console.log('High impact label detected, sending Slack notification');
+    console.log('High impact label detected, sending Slack notifications');
     slackNotification(`:alert: High Impact PR has been opened: <${html_url}|#${number}: ${title}>.` +
       ` Please prioritize testing the proposed changes.`, process.env.SLACK_HIGH_IMPACT_PR_WEBHOOK);
+    slackNotification(`:alert: High Impact PR has been opened: <${html_url}|#${number}: ${title}>.` +
+      ` Please review the PR details promptly and raise any concerns or questions.`,
+      process.env.SLACK_MILO_UPDATES_WEBHOOK);
```

Comment:
I think it's time we allow the method to take an array of webhooks to distribute the message to

Reply:
I did think about this, but I concluded it's not a common case just yet. Even here, this is an interim solution until we can rely solely on the new channel.